napari/_qt/_tests/test_qt_provide_theme.py

```diff
@@ -83,10 +83,6 @@ class TestPlugin:
         def napari_experimental_provide_theme():
             return {name: theme}
 
-    # create instance of viewer to make sure
-    # registration and unregistration methods are called
-    viewer = make_napari_viewer()
```

Comment:
I don't fully understand why this change was made, but I have a bad feeling about it because now it doesn't seem to match the pattern used everywhere else in the napari tests.

```python
def test_something(make_napari_viewer, other_args):
    viewer = make_napari_viewer()
    # ... rest of test...
```

Reply:
Just me testing something. I will revert this!